packages/remark-shaku-code-annotate/src/index.ts

```diff
@@ -108,8 +108,8 @@ export const remarkShakuCodeAnnotate = (
                 // TODO
                 break;
               case "DirectiveHighlight":
-                const mark = shakuLine.config.mark;
-                switch (mark) {
+                const highlightMark = shakuLine.config.mark;
```

Comment:
there is no need to change this , it is not better, let's just wrap it in a block scope.
```
case 'a': {
  const mark = ...
  break
}
case 'b': {
  const mark = ...
  break
}
```

Reply:
The names here can be unified. Thank you for pointing that out!